scio-jdbc/src/main/scala/com/spotify/scio/jdbc/JdbcConnectionOptions.scala

```diff
@@ -35,5 +35,35 @@ final case class JdbcConnectionOptions(
   username: String,
   password: Option[String],
   connectionUrl: String,
+  cloudSqlInstanceConnectionName: Option[String],
```

Comment:
Added this field because:
- `com.spotify.scio.jdbc.CloudSqlOptions` has `--cloudSqlInstanceConnectionName` arg, so some users can use it
- want to avoid parsing URL if possible